SCClassLibrary/Common/Control/Server.sc

```diff
@@ -544,7 +544,8 @@ Server {
 		case
 		{ failString.asString.contains("already registered") } {
 			"% - already registered with clientID %.\n".postf(this, msg[3]);
-			statusWatcher.notified = true;
+			statusWatcher.prRecoverRemoteLogin(msg[3]);
```

Comment:
could use a comment - without looking I don't know what msg[3] is

Reply:
done.